services/datalinker/templates/vault-pull-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: ricoberger.de/v1alpha1
```

Comment:
This file should be named `vault-secrets.yaml` to match the other applications.

Reply:
done